Core.UnitTests/Implementation/PipelineSettingsTest.cs

```diff
@@ -26,10 +26,9 @@ public class PipelineSettingsTest
     [Test]
     public void Initialization ()
     {
-      var settings = new PipelineSettings (true, "keyFile", true, "assemblyDirectory", "assemblyName{counter}suffix", 2);
+      var settings = new PipelineSettings (true, "keyFile", "assemblyDirectory", "assemblyName{counter}suffix", 2);
```

Comment:
Please craete a dedicated breaking change ticket that we have removed binary serialization support which means proxy types gneerated by TypePipe can no longer be used with BinarySerialization